src/main/java/org/fundacionjala/trello/pages/menus/Header.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.trello.pages.menus;
+
+import org.fundacionjala.trello.pages.board.BoardForm;
+import org.fundacionjala.trello.pages.forms.FormPage;
+import org.fundacionjala.trello.pages.trello.PageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public final class Header extends PageObject {
+
+    private static final String CREATE_MENU_BUTTON = "button[data-test-id=\"header-create-menu-button\"]";
+    private static final String CREATE_BOARD_BUTTON = "button[data-test-id=\"header-create-board-button\"]";
+    private static final String CREATE_TEAM_BUTTON = "button[data-test-id=\"header-create-team-button\"]";
+    private static final String BOARDS_MENU_BUTTON = "button[data-test-id=\"header-boards-menu-button\"]";
+
+    @FindBy(css = CREATE_MENU_BUTTON)
+    private WebElement creationButton;
+
+    @FindBy(css = CREATE_BOARD_BUTTON)
+    private WebElement createBoardButton;
+
+    @FindBy(css = CREATE_TEAM_BUTTON)
+    private WebElement createTeamButton;
+
+    @FindBy(css = BOARDS_MENU_BUTTON)
+    private WebElement headerMenuBoards;
```

Comment:
It seems to be, those web elements are not used.

Reply:
Yes, but I need them when I apply the step validations.